frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -240,17 +244,27 @@ export default {
     /**
      * Update user and emit success/fail event.
      */
-    async updateUser() {
+    async updateUser(user) {
       try {
-        //TODO - complete when API is ready
-        //this.user.organizationId = this.userInfo.organizationId
-        //const response = await ApiService.apiAxios.post(ApiRoutes.USER + '/update', this.user)
+        const response = await ApiService.apiAxios.post(ApiRoutes.USER + '/update', this.user)
         this.$emit('update-success-event', true)
       } catch (error) {
         this.$emit('update-success-event', false, error)
       }
     },
 
+    /**
+     * Get the list of facilities by contact id. If onlyWithPortalAccess is true, only return facilities with portal access.
+     */
+    async getUserFacilities(contactId, onlyWithPortalAccess) {
+      try {
+        const res = await ApiService.apiAxios.get(ApiRoutes.USER_FACILITIES + '/' + onlyWithPortalAccess + '/' + contactId)
```

Comment:
I would expect a URL of this sort to be something like:

/users/:contactId/facilities?onlyWithPortalAccess=true|false

It works for now but we should probably clean it up later.

Reply:
To review with team.